ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/MigrationsTest.java

```diff
@@ -412,6 +500,39 @@ private static SourceDescription describeSource(final String name) {
     return entity.getSourceDescription();
   }
 
+  private static int tableCount() {
```

Comment:
Rename this method to have the following signature:
```suggestion
  private static void assertTableCount(final int count){
```
and similarly for `connectorCount()` and `typeCount()` below, to avoid confusion.